settings.gradle

```diff
@@ -104,6 +104,7 @@ include 'data-prepper-core'
 include 'data-prepper-main'
 include 'data-prepper-pipeline-parser'
 include 'data-prepper-plugin-framework'
+include 'data-prepper-plugin-schema'
```

Comment:
Let's rename this to `data-prepper-plugin-schema-cli`. We may wish to split the schema out into a separate library in the future.